Products/CMFPlone/interfaces/controlpanel.py

```diff
@@ -1467,10 +1474,12 @@ class ISocialMediaSchema(Interface):
         default=True)
 
     twitter_username = schema.ASCIILine(
-        title=_(u'Twitter Username'),
-        description=_(u'To identify things like Twitter Cards. Do not include the \'@\' prefix character.'),
+        title=_(u'Twitter username'),
+        description=_(u'To identify things like Twitter Cards. Do not include the "@" prefix character.'),  # noqa: E501
```

Comment:
As said above, splitting a string into two lines is not that much of a problem, rather than adding noqa everywhere.